packages/@creature-chess/tools/src/inputs/components/cardShop.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
```

Comment:
rename this (and possibly other) files so as to be different from any existing modules - i.e. devCardShop